File: C/2309.c
#include <stdio.h>

int indexOf(int val);

int valoresCartas[] = {4, 5, 6, 7, 12, 11, 13, 1, 2, 3};

int main(){

    int n, adalberto = 0, bernadete = 0;

    scanf("%i", &n);

    for(int i = 0; i < n; i++){

        int a1, a2, a3, b1, b2, b3, rodadasA = 0, rodadasB = 0;

        scanf("%i %i %i %i %i %i", &a1, &a2, &a3, &b1, &b2, &b3);

        if(indexOf(a1) < indexOf(b1)){
            rodadasB += 1;
        }else{
            rodadasA += 1;
        }

        if(indexOf(a2) < indexOf(b2)){
            rodadasB += 1;
        }else{
            rodadasA += 1;
        }

        if(indexOf(a3) < indexOf(b3)){
            rodadasB += 1;
        }else{
            rodadasA += 1;
        }

        if(rodadasA >= rodadasB){
            adalberto += 1;
        }else{
            bernadete += 1;
        }
    }

    printf("%i %i\n", adalberto, bernadete);

    return 0;
}

int indexOf(int val){
    for (int i = 0; i < 10; i++){
        if (valoresCartas[i] == val){
            return i;
        }
    }
    return -1;
}

File: Python/2309.py
n = int(input())

adalberto, bernadete = 0, 0

valoresCartas = [4, 5, 6, 7, 12, 11, 13, 1, 2, 3]

for i in range(n):
    a1, a2, a3, b1, b2, b3 = map(int, input().split())
    
    rodadasA, rodadasB = 0, 0
    
    if valoresCartas.index(a1) < valoresCartas.index(b1):
        rodadasB += 1
    else:
        rodadasA += 1
    
    if valoresCartas.index(a2) < valoresCartas.index(b2):
        rodadasB += 1
    else:
        rodadasA += 1
        
    if valoresCartas.index(a3) < valoresCartas.index(b3):
        rodadasB += 1
    else:
        rodadasA += 1
        
    if rodadasA >= rodadasB:
        adalberto += 1
    else:
        bernadete += 1
        
print(adalberto, bernadete)